docs/source/daml/code-snippets/Proposal.daml

```diff
@@ -0,0 +1,22 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- A generic proposal workflow using generic templates.
+daml 1.2
+module Proposal where
+
+import DA.List
+
+-- start proposal body
+template Template t => Proposal t
+  with
+    asset : t
+    receivers : [Party]
+  where
+    signatory (signatory asset \\ receivers)
```

Comment:
It isn’t clear to me that removing the receivers is the right thing to do. I guess you want to prevent people from making a proposal that they can accept themselves? Maybe not a super useful thing to do but I also don’t see why it’s problematic. In any case, this deserves to be mentioned in the docs.

Reply:
Isn't the idea that you split the set of signatories of the proposed asset into two subsets. The _proposers_ which need to sign the proposal and the _receivers_. Here, we just compute the proposers from the signatories of the proposed asset and receivers. This makes a lot of sense to me.

Reply:
Thanks for the explanation, that does make sense to me as well but it should still be pointed out in the docs.